bucket/kryptor.json

```diff
@@ -0,0 +1,23 @@
+{
+    "homepage": "https://www.kryptor.co.uk/",
+    "description": "A simple, modern, and secure encryption and signing tool.",

```

Comment:
```suggestion
    "description": "Open source file encryption and signing tool.",
```

Reply:
Is this change necessary? I've used the same style as the [age](https://github.com/ScoopInstaller/Extras/commit/9287590c8efbb5a44cc43e5b5842d9a3a4872ed3) description.

Reply:
I took the description from your homepage itself. I feel it describes the software more succinctly:

![image](https://user-images.githubusercontent.com/46838874/150106408-66931b8e-98b9-473a-9099-4a69715e2d4c.png)

Reply:
Ok I'll go ahead and change it. Thanks for the explanation.